package.json

```diff
@@ -14,9 +14,9 @@
     "format": "yarn prettier --write",
     "check-format": "yarn prettier --list-different",
     "test": "jest --verbose",
-    "dev": "ts-node-dev --respawn --transpile-only ./samples/mediator.ts",
-    "prod:start": "node ./build/samples/mediator.js",
-    "prod:start-ws": "node ./build/samples/mediator-ws.js",
+    "dev": "ts-node-dev --respawn --transpile-only ./test/mediator.ts",
```

Comment:
Maybe we can use the same format as the rest of the codebase?

So `__tests__`?

Reply:
Hmm, not sure about that. To me, using `__tests__` has a kind of different purpose. In that case you're collocating tests with your source code files, then it's good to have this convention to express the difference. But at the highest level of hierarchy, you expect various types of folders. Maybe, what I was considering as an alternative is `e2e` or `e2e-tests` 🤷‍♂️

Reply:
I get your point. Ok. Maybe `tests` (plural) in that case? More in line with the rest of the codebase

Reply:
Agree 👍  changed